src/PrestaShopBundle/Command/UpdateSchemaCommand.php

```diff
@@ -218,7 +219,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 throw ($e);
             }
         }
-        $conn->commit();
+        if (!$conn->getWrappedConnection() instanceof PDO || $conn->getWrappedConnection()->inTransaction()) {
```

Comment:
Is this a safeguard to only commit when the transaction is open? But isn't it necessarily open since we call `beginTransaction` at the beginning of the function

Reply:
It is because of [implicit commit statements](https://mariadb.com/kb/en/sql-statements-that-cause-an-implicit-commit/), the behavior changed in PHP 8 but yeah, that's a safeguard to only commit when the transaction is open